alt-integration-adapter-java/src/main/java/org/veriblock/protoservice/IVeriBlockSecurity.java

```diff
@@ -50,4 +52,8 @@
     public ValidationResult setAltChainParametersConfig(AltChainParametersConfig config);
 
     public ValidationResult savePoPTransactionData(PoPTransactionData popTx, AltChainBlock containingBlock, AltChainBlock endorsedBlock);
+
+    public Pair<ValidationResult, List<VBlakeHash>> getLastKnownVBKBlocks(int maxBlockCount);
```

Comment:
We could use the VeriBlockBlock structure instead of hash.

Reply:
And that structure lacks getHash() on the client side which is what getpopdata has to return. Before GRPC migration, this api was indeed returning blocks.

Reply:
Indeed. We should make this Hash related API as well. Will make a ticket.